SDK/Implement/SubWallet.h

```diff
@@ -161,6 +161,7 @@ namespace Elastos {
 													  const std::string &status,
 													  const nlohmann::json &desc,
 													  uint32_t confirms);
+			void removeDuplicatedPrograms(const TransactionPtr &transaction);
```

Comment:
This method can be protected

Reply:
this method is already protected

Reply:
Move this function to class Transaction is better ?